test/rh-operators/metric_test.go

```diff
@@ -0,0 +1,88 @@
+package rhoperators
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	pclient "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/client"
+	psVersioned "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/client/clientset/versioned"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	kubeconfig = os.Getenv("KUBECONFIG")
+	testlog    = logrus.New()
+)
+
+// TestRHOperators installs all Red Hat Operators available on the cluster and see that they install successfully.
```

Comment:
I've always been dubious about the value this test adds b/c:

- It takes a dependency on content we don't own, the redhat-operators Catalog; its correctness directly affects whether our tests pass
- That external content can change irrespective of changes to OLM, making test outcomes become non-deterministic within the scope of this repository
- OLM should already have more generic e2e tests that catch regressions that would preclude previously installable content

Simply put, I think we should seriously consider dropping this test -- and the overhead that comes with maintaining it -- entirely.